dumper-integration-tests/redshift-tests/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'java'
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release"
+    }
+}
+
+dependencies {
+    implementation 'org.codehaus.groovy:groovy-all:3.0.5'
+    implementation 'org.testng:testng:7.5'
```

Comment:
`dumper` uses JUnit:
https://github.com/google/dwh-migration-tools/blob/844447c4fe60504580e816bae621b9e204ed75d4/dumper/buildSrc/src/main/groovy/dwh-migration-dumper.java-common-conventions.gradle#L110

Is there any specific need for `testng`?

Reply:
Yes, it's way easier to configure test suites, pass arguments and run in parallel if it's needed.
Also extended scope of `@Before<>` annotations and `@DataProvider` come very handy.
JUint works well with Unit tests and TestNG with integration.
Unless Junit is a strong requirement, I'd go with TestNG.